### ISSUE TITLE
Should we store assets in a different repository?

### ISSUE THREAD

**Issue by Sid-Bhatia-0** [JuliaReinforcementLearning/GridWorlds.jl#142]
@findmyway 

The total repo size of `GridWorlds.jl` is already around 55MB.
I was wondering if we should store assets in a different repository `GridWorldsAssets` and only use links in this repository. I understand that this might not be a very good practice in general. It is likely that we will add and modify the gifs in the future. And as the number of environments increases, modifications become costly for gifs.

It would be nice to freely experiment with different images & gifs, and not worry about the repo size drastically increasing.

**Comment by findmyway** [comment]
![empty_room_directed](https://user-images.githubusercontent.com/5612003/121120122-89837f00-c84f-11eb-8c45-337021d78498.gif)

Or maybe simply upload the image to github and use the generated link instead?
(We'd better to clean up the uploaded gif. 55M is way to large to this project.)

**Comment by Sid-Bhatia-0** [comment]
>Or maybe simply upload the image to github and use the generated link instead?

Good idea! We can have dedicated issues for uploading assets corresponding to each environment.

And once I finish doing this for all the current assets, we can use a tool like https://rtyley.github.io/bfg-repo-cleaner/ to delete those assets from the git history to significantly reduce the repo size.
